iOS/MMIWG2S/ViewControllers/IntroViewController.swift

```diff
@@ -0,0 +1,219 @@
+        self.titlevc = title
+        self.background = background
+        self.page = page
+        self.gradient = gradient
+        self.mmiwtext = mmiwtext
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+    
+        super.viewDidLoad()
+        let image = background
+        let imageView = UIImageView(image:image)
+        imageView.frame = view.frame
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        view.addSubview(imageView)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            imageView.topAnchor.constraint(equalTo: view.topAnchor),
+            imageView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1),
+            imageView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            imageView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+        
+        
+        let gradient = self.gradient
+        let gradientView = UIImageView(image:gradient)
+        gradientView.frame = view.frame
+        gradientView.contentMode = .scaleAspectFill
+        gradientView.clipsToBounds = true
+        view.addSubview(gradientView)
+        gradientView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            gradientView.topAnchor.constraint(equalTo: view.topAnchor),
+            gradientView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1),
+            gradientView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            gradientView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+
+        setupUI()
+    }
+    
+    func setupUI() {
+        
+        //Setup UI
+        let labelMMIW = UILabel(frame:CGRect(x:0,y:0,width:50,height:21))
+        let labelMMIWInfo = UILabel(frame:CGRect(x:0,y:0,width:200,height:200))
+        let buttonNext = UIButton(frame:CGRect(x:0,y:0, width:30, height:17))
+ 
+        let spacerView1 = UIView()
+        let spacerView2 = UIView()
+
+        labelMMIW.translatesAutoresizingMaskIntoConstraints = false
+        spacerView1.translatesAutoresizingMaskIntoConstraints = false
+
+        labelMMIWInfo.translatesAutoresizingMaskIntoConstraints = false
+        spacerView2.translatesAutoresizingMaskIntoConstraints = false
+
+        buttonNext.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(labelMMIW)
+        view.addSubview(labelMMIWInfo)
+        view.addSubview(spacerView1)
+
+        pc1View.translatesAutoresizingMaskIntoConstraints = false
+        pc2View.translatesAutoresizingMaskIntoConstraints = false
+        pc3View.translatesAutoresizingMaskIntoConstraints = false
+        pc4View.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
please move these views into an array. Something like `pageControlViews`? Then we could simply say
```
pageControlViews.forEach { 
    $0.backgroundColor = .white
    $0.translatesAutoresizingMaskIntoConstraints = false 
}
```

This will save a lot of lines and _more importantly_ it will be more robust code. Items like this should be in a set with an abstract count. That way if we were to add or remove one, it's a quick one line change. 

One thing to note is that if this approach is taken, it's prudent to have a guard at the top to ensure that `page  < pageControlViews.count`.

Reply:
Sent you a Slack message. about this one. I get an error using the gaurd statement something about it being out of scope.  It doesn't change color.